config.example.yaml

```diff
@@ -119,7 +120,22 @@ templates:
                   get:
                     backend_request:
                       uri: http://graphoid.wikimedia.org/{domain}/v1/png/{title}/{revision}/{graph_id}
-                      body: '{$.request.body}'
+
+      /{module:mathoid}:
+        x-modules:
+          - name: simple_service
+            version: 1.0.0
+            type: file
+            options:
+              paths:
+                /v1/{format}/{formula}/{type}:
+                  get:
+                    backend_request:
+                      method: post
+                      uri: http://mathoid-tester.wmflabs.org/{format}
+                      body:
+                        q: '{$.request.params.formula}'
```

Comment:
@Pchelolo yup, it has to be URI-encoded, so we're safe.

@physikerwelt the lowest maximum URL length is 2KB (old IE & IIS), modern browsers and servers accept arbitrary lengths, so we should be good here. Do you know of any real-world case where the formula would be longer than that?